sonar-go-checks/src/test/resources/checks/DuplicatedFunctionImplementationCheck/duplicated_function_implementation.go

```diff
@@ -0,0 +1,89 @@
+package DuplicatedFunctionImplementationCheck
+
+func foobar() {}
+func foo_bar () int {}  // Compliant - has no line
+
+func smallF1() {
+  foo = 1;
```

Comment:
```suggestion
  foo := 1;
```

The declaration and initialization requires `:=` instead of simple `=`